lib/quantum.ex

```diff
@@ -1,160 +1,202 @@
 defmodule Quantum do
+  @moduledoc """
+  Defines a quantum runner.
 
-  @moduledoc "A cron-like job scheduler"
+  When used, the quantum runner expects the `:otp_app` as option.
+  The `:otp_app` should point to an OTP application that has
+  the quantum rinner configuration. For example, the quantum runner:
 
-  alias Quantum.Job
-  alias Quantum.Normalizer
-  alias Quantum.Timer
+      defmodule MyApp.Quantum do
```

Comment:
I think this can be `MyApp.Job/Executer/Runner/Worker` but not just `MyApp.Quantum`, it's confusing what `Quantum` means. It's clear (relatively speaking) what `MyApp.Repo` means. And it can `use` the according `Quantum.Executer/Whatever` module. And `Quantum` itself can be used to contain client functions if necessary.